clients/admin-ui/src/features/user/types.ts

```diff
@@ -0,0 +1,94 @@
+export interface User {
+  id?: string;
+  first_name?: string;
+  last_name?: string;
+  username?: string;
+  password?: string;
+  created_at?: string;
+}
+
+export interface UserResponse {
+  data: {};
+  id: string;
+}
+
+export interface UsersResponse {
+  items: User[];
+  total: number;
+}
+
+export interface UsersListParams {
+  page: number;
+  size: number;
+  user: User;
+}
+
+export interface UserPasswordUpdate {
+  id: string | null;
+  old_password: string;
+  new_password: string;
+}
+
+export interface UserPermissionsUpdate {
+  id: string | null;
+  scopes: never[];
+}
+
+export interface UserPermissionsResponse {
+  data: {
+    id: string;
+  };
+  scope: string[];
+}
+
+export interface UserPrivileges {
+  privilege: string;
+  scope: string;
+}
+
+export const userPrivilegesArray: UserPrivileges[] = [
+  {
+    privilege: 'View subject requests',
+    scope: 'privacy-request:read',
```

Comment:
I think one day, not necessarily in this PR, we will want to pull these out as constants, such as like what is in the backend https://github.com/ethyca/fidesops/blob/user-management/src/fidesops/api/v1/scope_registry.py so that it'll be a lot easier in code to refer to the permissions